### ISSUE TITLE
[QUESTION] 

### ISSUE THREAD

**Issue by jd-s** [bibletime/bibletime#447]
**Background**
I am creating some material out of PD material, or material without license. Some of them are lexicons and dictionaries, some of them are books. They are shown as intended in other software like Xiphos, but BibleTime simply ignores all the layout.  

I wonder what the `correct' way to do this is, especially if stuff is working in other applications. I also like BibleTime very much and would love to see this working, also because more material will help to convince other people to use FOSS.

I generate books from osisText, dictionaries from tei with entryFree tags.

Thank you very much. 

**Question**

What is the correct way to manage the following things?

***Bible references***
``<reference osisRef="Rom.1.18">Röm 1,18</reference>`` gets ignored by BibleTime in books.
``<reference osisRef="Jes.48.12">48,12</reference>``gets ignored in dictionaries. 

***References in dictionaries***

``<xr type="see">→ <ref target="Abihu">Abihu</ref></xr>`` does not show up.

***Footnotes***

The following gets ignored by BibleTime:

```<note type="x-footnote">1 So Josef Ernst in einer Rezension in ThLZ 115,1990,593</note>```

***Figures***

In dictionaries, BibeTime ignores ``<graphic url="aundo.png" />``, in books it ignores ``<figure src="cover.jpg"/>``, while ``<figure src="21.jpg"/>`` is shown in maps. 

***Tables***

The following is ignored by BibleTime:
```
<table>
   <row>
      <cell>Liebe<note type="x-footnote">65 Kein Brief im corpus Paulinum redet quantitativ so häufig von ἀγάπη/ἀγαπνusw. wie Eph (Hinweis von R. Fuchs).</note></cell>
      <cell>I Tim 1,5  </cell>
      <cell>Eph 1,6; 4,15; 6,23f. u.ö. </cell>
      <cell>Apk 2,4  </cell>
   </row>
   <row>
      <cell> Umkehr/Sendung</cell>
      <cell> I Tim 1,12-17  </cell>
      <cell> Eph 2,1-7/3,5-10 </cell>
      <cell> Apk 2,5 </cell>
   </row>
   <row>
      <cell> Gebet</cell>
      <cell>  I Tim 2,1-10 </cell>
      <cell> Eph 6,18-20 </cell>
      <cell>  </cell>
   </row>
   <row>
      <cell> Frauen & Männer </cell>
      <cell> I Tim 2,8-15 </cell>
      <cell> Eph 5,22-33 </cell>
      <cell>  (Apk 2,6?)</cell>
   </row>
   <row>
      <cell>Alkoholgenuss </cell>
      <cell> I Tim 3,3; 5,23 </cell>
      <cell> Eph 5,18 </cell>
      <cell>  </cell>
   </row>
   <row>
      <cell> Generationenfrage</cell>
      <cell>  I Tim 5,1-8</cell>
      <cell> Eph 6,1-4 </cell>
      <cell>  </cell>
   </row>
   <row>
      <cell> Sklaven und Herren</cell>
      <cell> I Tim 6,1f. </cell>
      <cell> Eph 6,5-9 </cell>
      <cell>  </cell>
   </row>
   </table>
```

**Comment by gholmlund** [comment]
I don't believe anyone on the BibleTime team has a deep knowledge of OSIS. You might want to check with crosswire. Maybe there is a validation method for OSIS texts. 

If you would attach a working module with these problems, perhaps we can track down the issue.

**Comment by jd-s** [comment]
Great, thank you. 

I have also tracked down some issues with the lexdict modules. Please find the test module at the end of this text.

--- 

Here is the first entry, which contains text like ``<reference osisRef="2Sam.13.22">2Sam13,22</reference>``or ``<graphic url="aundo.png" />``. Here is screenshot:

![dict1](https://github.com/bibletime/bibletime/assets/25948654/1e603772-c1f0-4521-a6e1-43d4f45d03d7)

All references get ignored, and the line break ``<lb/>`` at the end behaves strange: The Image should be displayed after a line break. In addition, the ``<form type="headword">"A UND O</form>`` results in the fact that it is displayed twice.

---

The second entry also behaves strange. ``<hi type="italic">`` and ``<hi type="bold">`` get ignored, which leads - together with the headword - in a nearly unreadable mass of text in the beginning. Here, the image gets displayed after a line break, but the text after the image is not displayed. 

![dict2](https://github.com/bibletime/bibletime/assets/25948654/b3df1d9f-3773-4c5b-890d-48e412082822)

--

I am not sure, if this is related to the other issue.

Here is the file: 
[testdic.zip](https://github.com/bibletime/bibletime/files/14318823/testdic.zip)
